C/services/south/south.cpp

```diff
@@ -774,6 +779,83 @@ void SouthService::start(string& coreAddress, unsigned short corePort)
 	logger->info("South service shutdown %s completed", m_dryRun ? "from dry run " : "");
 }
 
+void SouthService::getResourceLimit()
+{
+	// Update the resource limit configuration
+	try 
+	{
+		// Parse the service buffering type
+		std::string serviceBuffering = m_configResourceLimit.getValue("serviceBuffering");
+		if (serviceBuffering == "Unlimited") 
+		{
+			m_serviceBufferingType = ServiceBufferingType::UNLIMITED;
+		} 
+		else if (serviceBuffering == "Limited") 
+		{
+			m_serviceBufferingType = ServiceBufferingType::LIMITED;
+		} 
+		else
+		{
+			Logger::getLogger()->error("Invalid serviceBuffering value: %s. Using default: UNLIMITED.", serviceBuffering.c_str());
```

Comment:
SERVICE_BUFFER_BUFFER_TYPE_DEFAULT instead of using a hard-coded value of UNLIMITED